data-raw/gostic_gt_pmf.R

```diff
@@ -0,0 +1,21 @@
+# E and I compartments both with exponentially distributed residence times
+# with a mean of 4 days.
+shape <- 2
+rate <- 1 / 4
+
+gostic_gt_pmf <- primarycensoreddist::dpcens(0:26,
+  pgamma,
+  shape = shape,
+  rate = rate,
+  D = 27
+) # v0.4.0
+
+# Drop first element because GI can't have same-day transmission
+gostic_gt_pmf <- gostic_gt_pmf[2:27]
```

Comment:
I also thought about supporting functionality for this natively in `primarycensoreddist`. What do you think and do you think it should exist? Essentially it would look like right truncation support as well as left truncation support I think (i.e you would pass 1:26 and specify min_D or similar and it would handle it).

Reply:
in some sense it could intoduce a API change (but I could handle with lifecycle) as maybe D should become D_lower and D_upperr or potentially even a 2 length vector (i.e (0, Inf) by default and then here (1, 27). If any of this is of interest maybe open a issue and we can push the discussion forward?

Reply:
https://github.com/epinowcast/primarycensoreddist/issues/63